src/poetry/core/json/schemas/project-schema.json

```diff
@@ -148,6 +148,22 @@
         "type": "string"
       }
     },
+    "import-names": {
+      "title": "Import names",
+      "description": "Import names that the project exclusively provides when installed.",
+      "type": "array",
+      "items": {
+        "type": "string"
```

Comment:
I wonder if we should be stricter here (cf https://www.schemastore.org/pyproject.json) or if we should do the validation that these are valid Python identifiers in `factory.py`?

Reply:
I've updated the schema for now.